integration-tests/features/stack_analyses_v2.feature

```diff
@@ -1,330 +1,181 @@
-
-  @requires_authorization_token @skip
-  Scenario: Check the analyzed dependencies for the exact version (arbitrary equality)
-    Given System is running
-    When I acquire the authorization token
-    Then I should get the proper authorization token
-    When I send Python package manifest requirements_click_arbitrary_equality.txt to stack analysis version 2 with authorization token
-    Then I should get 200 status code
-    When I wait for stack analysis version 2 to finish with authorization token
-    Then I should get 200 status code
-    Then I should find the value click under the path result/0/user_stack_info/analyzed_dependencies/0/package in the JSON response
-    Then I should find the value 6.7 under the path result/0/user_stack_info/analyzed_dependencies/0/version in the JSON response
-
-  @requires_authorization_token @skip
-  Scenario: Check the analyzed dependencies for older version of click package
-    Given System is running
-    When I acquire the authorization token
-    Then I should get the proper authorization token
-    When I send Python package manifest requirements_click_version_eq_5_0.txt to stack analysis version 2 with authorization token
-    Then I should get 200 status code
-    When I wait for stack analysis version 2 to finish with authorization token
-    Then I should get 200 status code
-    Then I should find the value click under the path result/0/user_stack_info/analyzed_dependencies/0/package in the JSON response
-    Then I should find the value 5.0 under the path result/0/user_stack_info/analyzed_dependencies/0/version in the JSON response
+    When I access the /api/v2/stack-analyses endpoint using the HTTP <method> method without user key
+    Then I should not get 200 status code
 
-  @requires_authorization_token @skip
-  Scenario: Check the analyzed dependencies for older version of click package with click>=5.0 in requirements
-    Given System is running
-    When I acquire the authorization token
-    Then I should get the proper authorization token
-    When I send Python package manifest requirements_click_version_ge_5_0.txt to stack analysis version 2 with authorization token
-    Then I should get 200 status code
-    When I wait for stack analysis version 2 to finish with authorization token
-    Then I should get 200 status code
-    Then I should find the value click under the path result/0/user_stack_info/analyzed_dependencies/0/package in the JSON response
-    Then I should find the value 6.7 under the path result/0/user_stack_info/analyzed_dependencies/0/version in the JSON response
+    Examples: HTTP methods
+     | method |
+     | GET    |
+     | HEAD   |
+     | PUT    |
+     | DELETE |
 
-  @requires_authorization_token @skip
-  Scenario: Check the analyzed dependencies for older version of click package with click>5.0 in requirements
+  @sav2-skip
```

Comment:
A couple of usages of this tag `@sav2-skip` ? Why we need it..?

Reply:
@sav2-skip is push mistake, I will be renaming all of them to @sav2, there will be no skip test cases in this file.